src/utils/promise.js

```diff
@@ -1,6 +1,8 @@
 // This is CommonJS because lie is an external dependency, so Rollup
 // can just ignore it.
-if (typeof Promise === 'undefined' && typeof require !== 'undefined') {
```

Comment:
`require` (or really `_dereq_`) will always be defined, so this check is not necessary ^